chapter_6/readme.md

```diff
@@ -0,0 +1,51 @@
+#Questions
+
+Questions
+
+##How did you resolve offenses detected by Rubocop?
+I went back through and made sure I was using single quotes opposed to double quotes.... That took 
+care of the offenses in lines 4, 19, and 20.
+I went in and made a .rubocop.yml file to fix the guardfile offense regarding line 19.
+ 
+These were the offenses I got...
+
+###Offenses:
+
+             ex6.rb:4:10: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
+             binary = "binary"
+             ^^^^^^^^
+             ex6.rb:19:5: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
+             w = "This is the left side of..."
+             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+             ex6.rb:20:5: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
+             e = "a string with a right side."
+             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+             Guardfile:19:9: C: Use // around regular expression.
+             watch(%r{.+\.rb$})
+             ^^^^^^^^^^^
+
+After writing the codeI went through the book and started creating the comments like
+I was asked to do, along with doing the other exercises that are in the Study Drills area.
+I went ahead and fixed all those offenses. I had 3 more. My comments were just too 
+long at first :) It wants you to have 80 characters or less on comments, so I just summed up my comments 
+to be a little shorter!
+
+## List the four places where a string is placed within another string. There are four places.
+
+      1         y = "Those who know #{binary}...
```

Comment:
Lines 35 and 36 are the same line, which would make it only three instances of string within a string. Please look through your file again and re-evaluate whether you think the author was lying or not